embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/api/ApiService.kt

```diff
@@ -74,5 +66,10 @@ internal interface ApiService {
      * @param crash the event message containing the crash
      */
     fun sendCrash(crash: EventMessage): Future<*>
-    fun sendSession(action: SerializationAction, onFinish: (() -> Unit)?): Future<*>?
+
+    /**
+     * Sends a session to the API. This can be either a v1 or v2 session - the implementation
+     * is responsible for routing the payload correctly.
+     */
+    fun sendSession(v2Payload: Boolean, action: SerializationAction, onFinish: (() -> Unit)?): Future<*>?
```

Comment:
Since this is no longer only controlling the payload but the endpoint as well, does it make sense to give the parameter a more generic name like `isV2`?